nexus-sdk/guides/setup.md

```diff
@@ -46,7 +46,7 @@ To install directly from the source using `cargo`, run:
 ```bash
 cargo install nexus-cli \
   --git https://github.com/talus-network/nexus-sdk \
-  --tag v0.1.0 \
+  --tag v0.2.0 \
```

Comment:
Where is this coming from? We don't have a `v0.2.0` released

Reply:
it's a merge into staging branch and it contains changes for 0.2.0